### PR TITLE
Points to Kubernetes docs for kubectl install

### PR DESCRIPTION
**Pull request by emanic**
I think it will be better to point people to the official Kubernetes documentation for the kubectl install.

## Description

Points people to the Kubernetes docs for the kubectl install. The procedure is much better documented there and this is also more maintainable.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: v2.4/getting-started/kubernetes/installation/vagrant/index.md [hunk 1]
```diff
@@ -76,21 +76,9 @@ And finally check that Docker is running on both hosts by running
...
-
-For Linux:
-
-```shell
-wget http://storage.googleapis.com/kubernetes-release/release/v1.7.0/bin/linux/amd64/kubectl
-chmod +x ./kubectl
-```
+Let's configure `kubectl` so you can access the cluster from your local machine. 
 
 Then, tell `kubectl` to use the Vagrant cluster we just created.
```
Comment (ozdanborne) [inline v2.4/getting-started/kubernetes/installation/vagrant/index.md]: This line is now a bit redundant with the one before it. Maybe remove this one?

**Comment by ozdanborne** [comment]
1 small nit. Then, can you make the same change to the files in `2.5/` and `master/` and this will be ready to merge

**Comment by bcreane** [comment]
one more nit - please squash the two commits.

**Comment by emanic** [comment]
@bcreane of course, good idea. I think we should be good now. 👍

**Comment by bcreane** [comment]
LGTM!

**Pull request merged**